ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -550,12 +557,23 @@ tests:
     cluster_profile: equinix-ocp-metal-qe
     env:
       AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      ENABLE_OTA_TEST: OCP-25473
```

Comment:
How about moving the case to OTA yaml file(in ci-operator/config/openshift/verification-tests/ dir)? Per previous discussion, if it's a checkpoint for OTA only and not suitable for general ci upgrade, we are recommended to host OTA dedicated test scenarios in OTA dedicated yaml instead of general CI yaml so that it will not break the upgrade ci on this profile. It looks like the retarget update way while there is already an upgrade ongoing is not a general update way, which meets this situation.

Reply:
@jiajliu Sure. 
BTW, The reason that why I choose this job is only because  baremetal job can be triggered by pj-rehearse, I tried many other platforms all of them got 'argument too long'.

Reply:
ah, got it. we could also add baremetal job or any other profile in OTA dedicated yaml according to your case's requirement